app/decorators/ext_management_system_decorator.rb

```diff
@@ -6,6 +6,10 @@ def fonticon
   end
 
   def listicon_image
-    "100/vendor-#{image_name}.png"
+    if kind_of?(ManageIQ::Providers::Hawkular::MiddlewareManager)
+      'vendor-hawkular'
+    else
+      "100/vendor-#{image_name}.png"
+    end
```

Comment:
@pilhuhn Thanks for pointing me to the SVGs. I'll update in a separate PR. We have a placeholder that you can copy for your PRs here: https://github.com/ManageIQ/manageiq/blob/master/app/assets/images/100/placeholder.png I'll also create an SVG version.  How should the WildFly image be named?